otlpserver/clievent.go

```diff
@@ -114,8 +108,14 @@ func NewCliEventFromSpan(span *tracepb.Span, ils *tracepb.InstrumentationLibrary
 	}
 
 	for _, attr := range span.GetAttributes() {
-		// TODO: break down by type so it doesn't return e.g. "int_value:99"
-		e.Attributes[attr.GetKey()] = attr.Value.String()
+		// TODO: this isn't great, there are ways it can cause mayhem, but
+		// should fine for known otlpserver use cases
+		val := attr.Value.String()
+		parts := strings.SplitN(val, "_value:", 2)
```

Comment:
Would it be safer to split on the colon and then later truncate "_value"?

Reply:
I think they're about the same. I want this to be fragile and break tests if some new representation shows up. Also I originally used `strings.Cut` but switched to SplitN to defer raising the minimum go version for otel-cli. It looked better with Cut, I thought about changing it, but chose fewer lines of code in the end.